src/App.js

```diff
@@ -1,8 +1,15 @@
 import './App.css';
-import Counter from "./components/counter/counter";
-import { SearchForm } from "./components/searchForm/searchForm";
-import { GenreSelect } from "./components/GenreSelect/GenreSelect";
+import  Movies from "./assets/movies.json";
+/* import Counter from './components/counter/counter';
```

Comment:
Please remove commented code

Reply:
I am using the commented code as a reference for future studies, for now it would be good to keep it.
But thanks for the note.